util.js

```diff
@@ -15,68 +15,38 @@
 'use strict'
 
 var Fs = require('fs')
-var Path = require('path')
-var _ = require('lodash')
-var xeno = require('xenotype')()
-
+var path = require('path')
```

Comment:
Can we be consistent with casing, either make `Fs` into `fs` or `path` into `Path`

I must prefer former (reserving pascal case for constructors, instead of designating core mods)